lib/active_model/serializer.rb

```diff
@@ -46,6 +47,10 @@ def self.inherited(base)
       super
     end
 
+    def self.type(type)
+      self._type = type
+    end
```

Comment:
Yeah, right, I see that instance should be as clean as possible to let users use their names for attributes.